llvm/test/tools/llvm-ml/cast.asm

```diff
@@ -0,0 +1,25 @@
+; RUN: llvm-ml -m64 -filetype=s %s /Fo - | FileCheck %s
```

Comment:
(I am travelling with limited computer access until May 4th, and my response time may be delayed..)

I am not familiar with llvm-ml parsing, does this exercise the object generation code path (MCObjectStreamer)? If not, this might not usefully test the X86AsmParser.cpp changes.